### PR TITLE
7.0 API Improvement

### PR DESCRIPTION
**Pull request by florian-dacosta**
Some change so it's easier to inherit the actual method, like it was discussed here on lauchpad : 
https://code.launchpad.net/~akretion-team/banking-addons/bank-statement-reconcile-70-api-improvement/+merge/197774

### CODE REVIEW

**Comment by bwrsandman** [comment]
Previously had 3x :+1: 
I'll add mine: :+1:

File: account_statement_base_completion/statement.py [hunk 1]
```diff
@@ -139,6 +139,12 @@ def _get_functions(self, cr, uid, context=None):
             ('get_from_label_and_partner_field', 'From line label (based on partner field)'),
             ('get_from_label_and_partner_name', 'From line label (based on partner name)')]
 
+    def __get_functions(self, cr, uid, context=None):
+        """
```
Comment (pedrobaeza) [inline account_statement_base_completion/statement.py]: This is not docstring compliant. Please put only one line.

File: account_statement_base_import/statement.py [hunk 2]
```diff
@@ -32,12 +32,15 @@
...
 
-    def get_import_type_selection(self, cr, uid, context=None):
+    def _get_import_type_selection(self, cr, uid, context=None):
         """This is the method to be inherited for adding the parser"""
         return [('generic_csvxls_so', 'Generic .csv/.xls based on SO Name')]
 
-    def _get_import_type_selection(self, cr, uid, context=None):
-        return self.get_import_type_selection(cr, uid, context=context)
+    def __get_import_type_selection(self, cr, uid, context=None):
+        """
```
Comment (pedrobaeza) [inline account_statement_base_import/statement.py]: Ídem

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**